jujugui/static/gui/src/app/app.js

```diff
@@ -746,6 +746,8 @@ YUI.add('juju-gui', function(Y) {
           username={username || 'anonymous'}
           charmstore={this.get('charmstore')} />,
         document.getElementById('charmbrowser-container'));
+      // The model name should not be visible when viewing the profile.
+      document.getElementById('environment-switcher').classList.add('hidden');
```

Comment:
This feels quite hacky, the header should be a component which we modify the state on. These types of modifications will become very difficult to find later on. Maybe land this as-is but do an immediate follow-up to create the breadcrumb into a component?